packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -113,11 +113,21 @@ function toTree(vnode) {
     }
     case HostText: // 6
       return node.memoizedProps;
+    case ForwardRef: {
+      return {
+        nodeType: 'function',
```

Comment:
hmm we should probably have a new type for this, but that would require much wider enzyme changes cc @aweary I think this is ok for now?